onnxruntime/python/tools/transformers/benchmark.py

```diff
@@ -99,18 +100,20 @@ def run_onnxruntime(use_gpu, provider, model_names, model_class, config_modifier
                 break
 
             input_names = all_input_names[:num_inputs]
+            args.model_type = MODELS[model_name][3]
+            fusion_options = FusionOptions.parse(args)
 
             if 'pt' in model_source:
                 with torch.no_grad():
                     onnx_model_file, is_valid_onnx_model, vocab_size, max_sequence_length = export_onnx_model_from_pt(
                         model_name, MODELS[model_name][1], MODELS[model_name][2], MODELS[model_name][3], model_class,
                         config_modifier, cache_dir, onnx_dir, input_names, use_gpu, precision, optimizer_info,
-                        validate_onnx, use_raw_attention_mask, overwrite, model_fusion_statistics)
+                        validate_onnx, use_raw_attention_mask, overwrite, model_fusion_statistics, fusion_options)
```

Comment:
might log some warning as fusion_options will only take effect when optimizer_info is set to by_script.
https://github.com/microsoft/onnxruntime/blob/e03b799b957a1828b2c470213ca3feac7edee78a/onnxruntime/python/tools/transformers/benchmark_helper.py#L34

Reply:
Added a warning.